net/haproxy/files/acme.hotplug

```diff
@@ -0,0 +1 @@
+[ renewed = "$ACTION" ] && /etc/init.d/haproxy reload
```

Comment:
This needs to also combine the key and fullchain cert into a single .pem file...

Reply:
Fixed.